common/metrics/service_wrapper.go

```diff
@@ -45,24 +45,24 @@ type (
 )
 
 const (
-	scopeNameDeprecateDomain                = "DeprecateDomain"
```

Comment:
This tally scope is also used by user code and we don't want their metrics to have this prefix.